go-controller/pkg/ovn/ovn.go

```diff
@@ -163,16 +176,17 @@ const (
 
 // NewOvnController creates a new OVN controller for creating logical network
 // infrastructure and policy
-func NewOvnController(kubeClient kubernetes.Interface, wf *factory.WatchFactory,
+func NewOvnController(kubeClient kubernetes.Interface, egressFirewallClient egressfirewallclientset.Interface, wf *factory.WatchFactory,
 	stopChan <-chan struct{}, addressSetFactory AddressSetFactory, ovnNBClient goovn.Client, ovnSBClient goovn.Client) *Controller {
 
 	if addressSetFactory == nil {
 		addressSetFactory = NewOvnAddressSetFactory()
 	}
 
 	return &Controller{
-		kube:                     &kube.Kube{KClient: kubeClient},
+		kube:                     &kube.Kube{KClient: kubeClient, EgressFirewallClient: egressFirewallClient},
 		watchFactory:             wf,
+		egressFirewallHandler:    nil,
```

Comment:
@JacobTanenbaum don't need to assign to nil here; it'll be automatically be nil if you don't set it.

Reply:
done

Reply:
done